src/vm/methodtablebuilder.cpp

```diff
@@ -10550,6 +10675,45 @@ MethodTableBuilder::SetupMethodTable2(
 #pragma warning(pop)
 #endif
 
+// Returns true if there is at least one default implementation for this interface method
+// We don't care about conflicts at this stage in order to avoid impact type load performance
+BOOL MethodTableBuilder::HasDefaultInterfaceImplementation(MethodDesc *pDeclMD)
```

Comment:
>HasDefaultInterfaceImplementation [](start = 25, length = 33)

This should also return false if the default interface #if is not enabled

Reply:
Fixed.